scripts/utils/tsDocParser.ts

```diff
@@ -58,17 +58,47 @@ const compilerOptions: CompilerOptions = {
 
 const Parser = withCompilerOptions(compilerOptions, TSDocOptions);
 
+interface ParseFunctionOptions {
+  /**
```

Comment:
are these changes related to poly default or should this be its own PR?

Reply:
Yeah, I made these changes to the parser to make sure we weren't publishing docs for the example compnents